contracts/src/bridge/Bridge.sol

```diff
@@ -32,15 +32,23 @@ contract Bridge is OwnableUpgradeable, DelegateCallAware, IBridge {
     address[] public allowedInboxList;
     address[] public allowedOutboxList;
 
-    address public override activeOutbox;
+    address private _activeOutbox;
 
     /// @dev Accumulator for delayed inbox messages; tail represents hash of the current state; each element represents the inclusion of a new message.
     bytes32[] public override inboxAccs;
 
+    address private constant EMPTY_ACTIVEOUTBOX = address(type(uint160).max);
+
     function initialize() external initializer onlyDelegated {
+        _activeOutbox = EMPTY_ACTIVEOUTBOX;
         __Ownable_init();
     }
 
+    function activeOutbox() public view returns (address) {
+        if(_activeOutbox == EMPTY_ACTIVEOUTBOX) return address(uint160(0));
```

Comment:
We should add a require in the `setOutbox` method to ensure that `allowedOutboxesMap` is never true for `type(uint160).max` and `0`